coalib/coala_modes.py

```diff
@@ -92,3 +92,36 @@ def mode_format():
     _, exitcode, _ = run_coala(
             print_results=print_results_formatted)
     return exitcode
+
+
+def mode_stdin(console_printer, log_printer):
+    import functools
```

Comment:
Needs a test. Probably feeding it args and verifying the exitcode is sufficient.